NewsApp/NewsApp/Controller/SearchLanguageController.swift

```diff
@@ -8,22 +8,55 @@
 
 import Foundation
 import UIKit
+import FlagKit
 
 class SearchLanguageController: UIViewController {
+    @IBOutlet private weak var languagePicker: UIPickerView!
+    @IBOutlet private weak var languageLabel: UILabel!
+    @IBOutlet private weak var resetButton: UIButton!
+
     weak var delegate: SearchService?
+    var languageSelected: Language?
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        initLanguageController()
+    }
+
+    func initLanguageController() {
+        initLanguageLabel()
+        self.navigationItem.title = R.string.localizable.language()
+        guard let language = languageSelected else {
+            return
+        }
+        if let index = language.index {
+            languagePicker.selectRow(index, inComponent: 0, animated: false)
+        }
```

Comment:
Plutot préféré de coupler les tests dans un if let plutôt que guard let + if let

le guard let est à utiliser plutot en debut de fonction pour valider les inputs

Reply:
fix in "correction PR"